DuckDuckGo/Suggestion.swift

```diff
@@ -18,9 +18,22 @@
 //
 
 import Foundation
+import Core
 
-struct Suggestion {
+struct Suggestion: Decodable {
 
     let type: String
     let suggestion: String
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.singleValueContainer()
+        let dictionary = try container.decode([String: String].self)
+
+        if let key = dictionary.keys.first, let value = dictionary[key] {
```

Comment:
You could use `let entry = dictionary.first` and just reference key/value without dictionary lookup.

Reply:
Ah good call, thanks!